contrib/building.md

```diff
@@ -168,6 +169,14 @@ mklink /j assets "../../../../../data"
 
 Note: If you have cloned the repo previously make sure you remove your `node_modules` directory since it may contain stale dependencies which may cause the build to fail.
 
+### Visual Studio Code setup for C++
```

Comment:
Do you need the configuration mentioned in https://code.visualstudio.com/docs/cpp/config-clang-mac? Or does `clangd` handle it for you?

Reply:
I think the `clangd` extension should handle this, it's independent of the MS C++ extension as I understand it... I have the MS Intellisense disabled and it seems to work but if you could verify on your setup that would be great!